module/programs/helper.js

```diff
@@ -113,10 +127,11 @@ module.exports = class ProgramsHelper {
    * @method
    * @name create
    * @param {Array} data
+   * @param {Boolean} checkDate this is true for when its called via API calls
    * @returns {JSON} - create program.
    */
 
-  static create(data) {
+  static create(data, checkDate = false,token) {
```

Comment:
@praveenKDass why we need token? please remove if not required

Reply:
removed token @VISHNUDAS-tunerlabs